files/image_config/config-setup/config-setup.conf

```diff
@@ -0,0 +1,4 @@
+# conf file for config-setup
+# file: /etc/config-setup/config-setup.conf
+#
+KEEP_BASIC_TABLES='["MGMT_PORT","MGMT_INTERFACE","PASSW_HARDENING"]'
```

Comment:
What is the rationale behind choosing these tables? If you are saving everything w.r.t mgmt port please include MGMT_VRF_CONFIG

Reply:
We want to allow the user to preserve a basic set of configurations that he will likely always want to to preserve and settings necessary for the box to be reachable from the outside.
So user won't need to reconfigure his MGMT interface, ssh servers and users of the system ( in future).

Sure I add it to the list.